flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackendBuilder.java

```diff
@@ -118,7 +118,7 @@ public RocksDBKeyedStateBackendBuilder(
 		String operatorIdentifier,
 		ClassLoader userCodeClassLoader,
 		File instanceBasePath,
-		DBOptions dbOptions,
+		RocksDBResourceContainer optionsContainer,
```

Comment:
@carp84  changes in this file [have broken the continuous benchmarking](https://issues.apache.org/jira/browse/FLINK-15199) :( One thing is that they do not work blocking solving some release blockers, another thing is that now doing bisection including this commit will be painful.

If some class is used in the benchmarks, please try treating it as a more or less `Public` api (I think I will propose exactly this on the dev mailing list). It's best to preserve the compatibility somehow, and if that's not possible, at least please remember to adjust the benchmarks code.

Reply:
Thanks for the note @pnowojski , will prepare a follow up PR to prevent future changes from breaking the state-benchmark again.

Reply:
Follow up PR created: #10548